go/libkb/interfaces.go

```diff
@@ -441,3 +442,50 @@ type GregorListener interface {
 	PushHandler(handler GregorInBandMessageHandler)
 	PushFirehoseHandler(handler GregorFirehoseHandler)
 }
+
+type GlobalContextLite interface {
```

Comment:
I don't think we've found the right name yet.  Maybe let's change it when we hit a second implementation/use of it outside this case.  I guess we can always change via `gofmt` as long as it's all in the same repo